src/slapper/entities/SlapperEntity.php

```diff
@@ -58,7 +60,7 @@ protected function sendSpawnPacket(Player $player): void {
         $pk2->uuid = UUID::fromRandom();
         $pk2->username = $this->getDisplayName($player);
         $pk2->position = $this->asVector3()->add(0, static::HEIGHT);
-        $pk2->item = ItemFactory::get(ItemIds::AIR);
+        $pk2->item = ItemStackWrapper::legacy(Item::get(Item::AIR));
```

Comment:
Here, changing ItemFactory and ItemIds to Item is an unrelated change. Please revert it. Also, remove the added import for `pocketmine\item\Item`